instrumentation-api/src/main/java/io/opentelemetry/instrumentation/api/tracer/async/MethodSpanStrategyContextKey.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.api.tracer.async;
+
+import io.opentelemetry.context.ContextKey;
+
+final class MethodSpanStrategyContextKey {
```

Comment:
Why a separate class for this? Key can live in `MethodSpanStrategy`, right?

Reply:
It could, I was following the convention set by `Span`+`SpanContextKey` where the `ContextKey<T>` isn't publicly accessible, but if that doesn't matter I can make it a static final field on the `MethodSpanStrategy` interface instead.